front/tests/utils/api-setup.ts

```diff
@@ -89,16 +89,18 @@ export const postApiRequest = async <T>(
  * Send a DELETE request to the specified API endpoint.
  *
  * @param url - The API endpoint URL.
- * @returns {Promise<APIResponse>} - The response from the API.
+ * @returns {Promise<Response>} - The API response.
  */
-export const deleteApiRequest = async (
-  url: string,
-  errorMessage?: string
-): Promise<APIResponse> => {
-  const apiContext = await getApiContext();
-  const response = await apiContext.delete(url);
-  handleErrorResponse(response, errorMessage);
-  return response;
+export const deleteApiRequest = async (url: string): Promise<Response> => {
+  const fullUrl = `http://localhost:4000${url}`;
+
+  return fetch(fullUrl, {
+    method: 'DELETE',
+    headers: {
+      Accept: '*/*',
+      'Content-Type': 'application/json',
+    },
+  });
 };
```

Comment:
```suggestion
export const deleteApiRequest = async (urlExtend: string): Promise<Response> => {
  const fullUrl = `${baseUrl}${urlExtend}`;

  return fetch(fullUrl, {
    method: 'DELETE',
    headers: {
      Accept: '*/*',
      'Content-Type': 'application/json',
    },
  });
};
```

they add on top `const baseUrl = 'http://localhost:4000'` and set it in `getApiContext`